repo_compare/module_compare_html.sh

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
bash shebang must be on first line.
`pre-commit` / `shellcheck` catches this before commit. 😉

Reply:
Must have been an issue when I resolved a merge conflict earlier.  Should all pass shellcheck now